### ISSUE TITLE
NativeProviders.txt is out-of-date

### ISSUE THREAD

**Issue by tibel** [mathnet/mathnet-numerics#315]
The file is quite old and does not contain current information.
It think it should be removed completely.
Alternative would be to create a `README.md` in `src/NativeProviders` folder.

**Comment by tibel** [comment]
Also `Math.Net Master Slides.pptx` seems outdated.

**Comment by tibel** [comment]
Hmm, maybe the wiki (https://github.com/mathnet/mathnet-numerics/wiki) should be cleaned up too (or removed).

**Comment by cdrnet** [comment]
Yes, we should clean this up. All build-relevant information should go to [Building Math.NET Numerics](http://numerics.mathdotnet.com/Build.html).
